---
layout: page
title: Publications
permalink: /publications/
---

A list of my most important publications follows (To see all of them, please refer to my CV). Many of them are <a href="http://en.wikipedia.org/wiki/Open_access_%28publishing%29">Open Access</a>; a PDF is attached to the ones that are not.

<ul>
  <li><a href="#Journals">Journals</a></li>
  <li><a href="#Chapters">Chapters in books</a></li>
  <li><a href="#Conference">Conference proceedings</a></li>
  <li><a href="#Preprints">Preprints, Workshop Proceedings and other publications</a></li>
  <li><a href="#Etc.">Etc.</a></li>
</ul>

<h2 id="Journals">Journals</h2>

Mikel Val-Calvo, Mikel Egaña Aranguren, Juan Mulero-Hernández, Ginés Almagro-Hernández, Prashant Deshmukh, José Antonio Bernabé-Díaz, Paola Espinoza-Arias, José Luis Sánchez-Fernández, Juergen Mueller, Jesualdo Tomás Fernández-Breis. OntoGenix: Leveraging Large Language Models for enhanced ontology engineering from datasets. <em>Information Processing & Management</em> 2025: 62 (3). [<a href="https://doi.org/10.1016/j.ipm.2024.104042">Information Processing & Management</a>]

Mikel Egaña Aranguren, Mark D. Wilkinson. Enhanced reproducibility of SADI web service workflows with Galaxy and Docker. GigaScience 2015: 4(59). [<a href="https://gigascience.biomedcentral.com/articles/10.1186/s13742-015-0092-3">GigaScience</a>][<a href="https://addi.ehu.es/handle/10810/68964">ADDI</a>][<a href="https://doi.org/10.1186/s13742-015-0092-3"><img src="https://zenodo.org/badge/DOI/10.1186/s13742-015-0092-3.svg" alt="DOI"></a>]

Pawluczyk, M., Weiss, J., Links, M. G., Aranguren, M. E., Wilkinson, M. D., and Egea-Cortines, M. Quantitative evaluation of bias in PCR amplification and Next Generation Sequencing derived from metabarcoding samples. <em>Analytical and Bioanalytical Chemistry</em> 2015: 407(7):1841–1848. [<a href="http://link.springer.com/article/10.1007%2Fs00216-014-8435-y#">Analytical and Bioanalytical Chemistry</a>][<a href="pawluczyk_abc1.pdf">pdf</a>]

Alejandro Rodríguez González, Alison Callahan, José Cruz-Toledo, Adrian Garcia, Mikel Egaña Aranguren, Michel Dumontier, Mark D. Wilkinson. Automatically exposing OpenLifeData via SADI semantic Web Services. <em>Journal of Biomedical Semantics</em> 2014, 5(1):46+. [<a href="http://www.jbiomedsem.com/content/5/1/46/abstract">Journal of Biomedical Semantics</a>]

Mikel Egaña Aranguren, Alejandro Rodríguez González, Mark D. Wilkinson. Executing SADI services in Galaxy. <em>Journal of Biomedical Semantics</em> 2014, 5(1):42+. [<a href="http://www.jbiomedsem.com/content/5/1/42/abstract">Journal of Biomedical Semantics</a>][<a href="https://addi.ehu.es/handle/10810/15997">ADDI</a>][<a href="https://doi.org/10.5281/zenodo.13760079"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.13760079.svg" alt="DOI"></a>]

Mikel Egaña Aranguren, Jesualdo Tomás Fernández-Breis, Michel Dumontier. Special issue on Linked Data for Health Care and the Life Sciences (Editorial). <em>Semantic Web Journal</em> 2014, 5(2): 99-100. [<a href="egana-aranguren-et-al-2014-special-issue-on-linked-data-for-health-care-and-the-life-sciences.pdf">pdf</a>][<a href="https://content.iospress.com/articles/semantic-web/sw115">IOS press</a>][<a href="http://www.semantic-web-journal.net/content/special-issue-linked-data-health-care-and-life-sciences">Semantic Web Journal</a>]

José Antonio Miñarro-Giménez, Mikel Egaña Aranguren, Boris Villazón-Terrazas, Jesualdo Tomás Fernández Breis. Translational research combining orthologous genes and human diseases with the OGOLOD dataset. <em>Semantic Web Journal</em> 2014, 5(2): 145-149. [<a href="https://content.iospress.com/articles/semantic-web/sw109">IOS press</a>][<a href="http://www.semantic-web-journal.net/content/translational-research-combining-orthologous-genes-and-human-diseases-ogolod-dataset">Semantic Web Journal</a>]

Mikel Egaña Aranguren, Jesualdo Tomás Fernández-Breis, Erick Antezana, Chris Mungall, Alejandro Rodríguez González, Mark D. Wilkinson. OPPL-Galaxy, a Galaxy tool for enhancing ontology exploitation as part of bioinformatics workflows. <em>Journal of Biomedical Semantics</em> 2013, 4 (1):2+. [<a href="http://www.jbiomedsem.com/content/4/1/2/abstract">Journal of Biomedical Semantics</a>]

Astrid Duque-Ramos, Jesualdo T. Fernández-Breis, Miguela Iniesta, Michel Dumontier, Mikel Egaña Aranguren, Stefan Schulz, Nathalie Aussenac-Gilles, and Robert Stevens. Evaluation of the OQuaRE framework for ontology quality. <em>Expert Systems with Applications</em> 2013, 40(7):2696–2703. [<a href="http://www.sciencedirect.com/science/article/pii/S0957417412012146">Science Direct</a>][<a href="eswa8231.pdf">pdf</a>][<a href="http://hdl.handle.net/10810/74564">ADDI</a>][<a href="https://doi.org/10.1016/j.eswa.2012.11.004"><img src="https://zenodo.org/badge/DOI/10.1016/j.eswa.2012.11.004.svg" alt="DOI"></a>]

José Antonio Miñarro-Giménez, Mikel Egaña Aranguren, Boris Villazón-Terrazas and Jesualdo Tomás Fernández-Breis. Publishing Orthology and Diseases Information in the Linked Open Data cloud.<em> Current Bioinformatics</em> 2012, 7 (3), 255-266. [<a href="http://dx.doi.org/10.2174/157489312802460811">Bentham science</a>][<a href="currentbioinformatics-ogolod.pdf">pdf</a>][<a href="https://doi.org/10.2174/157489312802460811"><img src="https://zenodo.org/badge/DOI/10.2174/157489312802460811.svg" alt="DOI"></a>]

Vladimir Mironov, Erick Antezana, Mikel Egaña, Ward Blondé, Bernard De Baets, Robert Stevens and Martin Kuiper. Flexibility and utility of the cell cycle ontology. <em>Applied Ontology</em> 2011, 6 (3), 247-261. [<a href="http://dx.doi.org/10.3233/AO-2011-0097">IOS Press</a>][<a href="247-261.pdf">pdf</a>][<a href="http://hdl.handle.net/10810/69709">ADDI</a>][<a href="https://doi.org/10.3233/AO-2011-0097"><img src="https://zenodo.org/badge/DOI/10.3233/AO-2011-0097.svg" alt="DOI"></a>]

Jose Antonio Miñarro Gimenez; Mikel Egaña Aranguren, Rodrigo Martínez Béjar, Jesualdo Tomás Fernández Breis, Marisa Madrid. Semantic integration of information about orthologs and diseases: The OGO system. <em>Journal of Biomedical Informatics </em>2011, 44, 1020-1031. [<a href="http://dx.doi.org/10.1016/j.jbi.2011.08.002">Science Direct</a>][<a href="jbi-ogo.pdf">pdf</a>][<a href="http://hdl.handle.net/10810/68963">ADDI</a>][<a href="https://doi.org/10.1016/j.jbi.2011.08.002"><img src="https://zenodo.org/badge/DOI/10.1016/j.jbi.2011.08.002.svg" alt="DOI"></a>]

Erick Antezana, Ward Blondé, Mikel Egaña, Alistair Rutherford, Robert Stevens, Bernard De Baets, Vladimir Mironov, Martin Kuiper. BioGateway: a semantic systems biology tool for the life sciences. <em>BMC bioinformatics</em> 2009, 10(Suppl 10):S11. [<a href="http://www.biomedcentral.com/1471-2105/10/S10/S11">BMC bioinformatics</a>][<a href="http://hdl.handle.net/10810/69536">ADDI</a>][<a href="https://doi.org/10.1186/1471-2105-10-S10-S11"><img src="https://zenodo.org/badge/DOI/10.1186/1471-2105-10-S10-S11.svg" alt="DOI"></a>]

Erick Antezana, Mikel Egaña, Bernard De Baets, Ward Blondé, Aitzol Illarramendi, Iñaki Bilbao, Bernard De Baets, Robert Stevens, Vladimir Mironov and Martin Kuiper. The Cell Cycle Ontology: An application ontology for the representation and integrated analysis of the cell cycle process. <em>Genome Biology </em>2009, 10:R58. [<a href="https://doi.org/10.1186%2Fgb-2009-10-5-r58">Genome Biology</a>][<a href="http://hdl.handle.net/10810/68881">ADDI</a>][<a href="https://doi.org/10.1186/gb-2009-10-5-r58"><img src="https://zenodo.org/badge/DOI/10.1186/gb-2009-10-5-r58.svg" alt="DOI"></a>]

Mikel Egaña Aranguren, Erick Antezana, Martin Kuiper, Robert Stevens. Ontology Design Patterns for bio-ontologies: a case study on the Cell Cycle Ontology.<em> BMC bioinformatics</em> 2008, 9(Suppl 5):S1. [<a href="http://www.biomedcentral.com/1471-2105/9/S5/S1">BMC bioinformatics</a>][<a href="https://addi.ehu.es/handle/10810/68874">ADDI</a>][<a href="https://doi.org/10.1186/1471-2105-9-S5-S1"><img src="https://zenodo.org/badge/DOI/10.1186/1471-2105-9-S5-S1.svg" alt="DOI"></a>]

Mikel Egaña Aranguren, Chris Wroe, Carole Goble, Robert Stevens. <em>In situ</em> migration of handcrafted ontologies to Reason-able Forms. <em>Data &amp; Knowledge Engineering</em> 2008, 66, 147-162. [<a href="http://dx.doi.org/10.1016/j.datak.2008.02.002">Science Direct</a>][<a href="in_situ.pdf">pdf</a>][<a href="https://addi.ehu.es/handle/10810/68880">ADDI</a>][<a href="https://doi.org/10.1016/j.datak.2008.02.002"><img src="https://zenodo.org/badge/DOI/10.1016/j.datak.2008.02.002.svg" alt="DOI"></a>]

Erick Antezana, Mikel Egaña Aranguren, Bernard De Baets, Martin Kuiper, Vladimir Mironov. ONTO-PERL: An API supporting the development and analysis of bio-ontologies. <em>Bioinformatics</em> 2008, 24(6):885-887. [<a href="https://doi.org/10.1093/bioinformatics/btn042">Bioinformatics</a>][<a href="https://addi.ehu.es/handle/10810/69537">ADDI</a>][<a href="https://doi.org/10.1093/bioinformatics/btn042"><img src="https://zenodo.org/badge/DOI/10.1093/bioinformatics/btn042.svg" alt="DOI"></a>]

Mikel Egaña Aranguren, Sean Bechoffer, Phillip Lord, Ulrike Sattler and Robert Stevens. Understanding and using the meaning of statements in a bio-ontology: recasting the Gene Ontology in OWL. <em>BMC Bioinformatics</em> 2007, 8:57. [<a href="http://www.biomedcentral.com/1471-2105/8/57">BMC bioinformatics</a>][<a href="https://addi.ehu.es/handle/10810/68801">ADDI</a>][<a href="https://doi.org/10.1186/1471-2105-8-57"><img src="https://zenodo.org/badge/DOI/10.1186/1471-2105-8-57.svg" alt="DOI"></a>]

Robert Stevens, Mikel Egaña Aranguren, Katy Wolstencroft, Ulrike Sattler, Nick Drummond and Mathew Horridge. Using OWL to Model Biological Knowledge. <em>International Journal of Human Computer Studies</em> 2006, 65:7, 583-594. [<a href="http://dx.doi.org/10.1016/j.ijhcs.2007.03.006">Science Direct</a>][<a href="using_owl.pdf">pdf</a>][<a href="https://doi.org/10.1016/j.ijhcs.2007.03.006"><img src="https://zenodo.org/badge/DOI/10.1016/j.ijhcs.2007.03.006.svg" alt="DOI"></a>]

<h2 id="Chapters">Chapters in books</h2>

Mikel Egaña Aranguren, Robert Stevens, Erick Antezana, Jesualdo Tomás Fernández-Breis, Martin Kuiper, Vladimir Mironov. Technologies and best practices for building bio-ontologies. In Marco Ramoni and Gil Alterovitz (Eds), Knowledge-based Bioinformatics: From Analysis to Interpretation. ISBN: 978-0-470-74831-2. [<a href="https://www.wiley.com/en-us/Knowledge-Based+Bioinformatics%3A+From+Analysis+to+Interpretation-p-9781119995838">Wiley</a>][<a href="c03.pdf">pdf</a>]

<h2 id="Conference">Conference proceedings</h2>

Alejandro Rodríguez González, Marcos Martínez Romero, Mikel Egaña Aranguren, Mark D. Wilkinson. Nanopublishing clinical diagnoses: tracking diagnostic knowledge base content and utilization. 27th International Symposium on Computer-Based Medical Systems (CBMS 2014): 335-340. [<a href="https://doi.org/10.1109/CBMS.2014.82">IEEE Xplore</a>][<a href="4435a335.pdf">pdf</a>]

Alejandro Rodríguez Iglesias, Mikel Egaña Aranguren, Alejandro Rodríguez González, Mark D. Wilkinson. Plant-Pathogen Interactions Ontology (PPIO). International Work-Conference on Bioinformatics and Biomedical Engineering (IWBBIO 2013), pages 695-702. [<a href="http://iwbbio.ugr.es/papers/iwbbio_110.pdf">iwbbio.ugr.es</a>][<a href="iwbbio2013_submission_27.pdf">pdf</a>]

Mikel Egaña Aranguren, Jesualdo Tomás Fernández-Breis, Erick Antezana. OPPL-Galaxy: Enhancing ontology exploitation in Galaxy with OPPL.  In Proceedings of the 4th International Workshop on Semantic Web Applications and Tools for the Life Sciences, SWAT4LS 2011, pages 12–19, ACM 2012. [<a href="http://dx.doi.org/10.1145/2166896.2166903">ACM</a>][<a href="p12-aranguren.pdf">pdf</a>]

J.A. Miñarro-Giménez, M. Egaña Aranguren, F. García-Sánchez, J. T. Fernández-Breis. A Semantic Query Interface for the OGO Platform. ITBAM (DEXA) 2010, LNCS 6266, pp. 128-142. [<a href="https://doi.org/10.1007/978-3-642-15020-3_13">SpringerLink</a>][<a href="dexa.pdf">pdf</a>]

Mikel Egaña, Alan Rector, Robert Stevens, Erick Antezana. Applying Ontology Design Patterns in bio-ontologies. EKAW 2008, LNCS 5268, pp. 7-16. [<a href="https://doi.org/10.1007/978-3-540-87696-0_4">SpringerLink</a>][<a href="ekaw.pdf">pdf</a>]

<h2 id="Preprints">Preprints, Workshops and other publications</h2>

Joel Garcia-Escribano, Arkaitz Carbajo, Mikel Egaña Aranguren, Unai Lopez-Novoa. Using GPT to build a Project Management assistant for Jira environments. [<a href="https://arxiv.org/abs/2509.26014">Arxiv</a>][<a href="2509.26014v1.pdf">pdf</a>]

Aranguren, Mikel Egaña and Fernández Breis, Jesualdo Tomás and Balentzia, Bidane Leon and Garcia Castro, Alexander and Rompe, Markus, A Comprehensive View of Software Vulnerability Risks Through Enterprise Knowledge Graphs. [<a href="https://dx.doi.org/10.2139/ssrn.5388699">SSRN</a>][<a href="ssrn-5388699.pdf">pdf</a>]

Aranguren, M. E. (2024). The Transparent Relations Ontology (TRO): a vocabulary to describe conflicts of interest. [<a href="https://doi.org/10.48550/arXiv.2410.07154">Arxiv</a>][<a href="2410.07154v1.pdf">pdf</a>]

Aranguren, M. E., Breis, J. T. F., Castro, A. G., and Rompe, M. (2024). The Cybersecurity Ontology Network: the first building block for a comprehensive Cybersecurity Knowledge Graph. In OK4I - Ontologies and Knowledge Graphs for Industry (Co-located with FOIS), Enschede, Netherlands. [<a href="https://ceur-ws.org/Vol-3882/">CEUR workshops volume 3882</a>][<a href="Cybersecurity_Ontology_Network.pdf">pdf</a>]

Aranguren, M. E. (2015). Merging OpenLifeData with SADI services using Galaxy and Docker (DOI 10.1101/013615). BioRxiv, Cold Spring Harbor Labs. [<a href="http://dx.doi.org/10.1101/013615">Biorxiv</a>][<a href="013615v1.full.pdf">pdf</a>]

Marshall, M., Boyce, R., Deus, H., Zhao, J., Willighagen, E., Samwald, M., Pichler, E., Hajagos, J., Aranguren, M., Miller, M., Prudhommeaux, E., Dumontier, M., and Stephens, S. (2012). Health Care and Life Science (HCLS) Linked Data Guide (http://www.w3.org/2001/sw/hcls/notes/hcls-rdf-guide/)

Horridge, M., Aranguren, M., Mortensen, J., Musen, M., and Noy, N. (2012). Ontology Design Pattern Language Expressivity Requirements. In WOP (Co-located with ISWC), Boston, USA. [<a href="paper3.pdf">pdf</a>]

Gimenez, J., Aranguren, M., and Tomas, J. (2012). NCBO-Galaxy: bridging the BioPortal web services and the Galaxy platform. In ICBO, Graz, Austria. [<a href="demo_3.pdf">pdf</a>]

Fernandez-Breis, J., Aranguren, M., and Stevens, R. (2009). A quality evaluation framework for bio-ontologies. In ICBO, Buffalo, USA. [<a href="npre.2009.3479.1.pdf">pdf</a>]

Aranguren, M., Stevens, R., and Antezana, E. (2008b). Transforming the Axiomisation of Ontologies: The Ontology Pre-Processor Language. In OWL Experiences and Directions (OWLEd), Washington DC, USA.

Iannone, L., Egana, M., Rector, A., and Stevens, R. (2008). Augmenting the expressivity of the ontology pre-processor language. In OWL Experiences and Directions (OWLEd), co-located with ISWC, Karlsruhe, Germany. [<a href="Augmenting_the_Expressivity_of_the_Ontology_Pre-Pr.pdf)">pdf</a>]

Antezana, E., Blondé, W., Egana, M., Rutherford, A., Stevens, R., De Baets, B., Mironov, V., and Kuiper, M. (2008a). Structuring the life science resourceome for semantic systems biology: lessons from the BioGateway Project. In Proceedings of the Workshop on Semantic Web Applications and Tools for Life Sciences (SWAT4LS): November 28, 2008; Edinburgh, United Kingdom. [<a href="SWAT4LS_2008_paper.pdf">pdf</a>]

<h2 id="Etc.">Etc.</h2>

<strong>English</strong>

PhD thesis: Role and Application of Ontology Design Patterns in Bio-ontologies. Mikel Egaña Aranguren, 2009, University of Manchester, Faculty of Engineering and Physical Sciences, School of Computer Science. [<a href="thesis.pdf">pdf</a>]

MSc thesis: Improving the structure of the Gene Ontology. Mikel Egaña Aranguren, 2004, University of Manchester, Faculty of Engineering and Physical Sciences, School of Biological Sciences. [<a href="msc_thesis.pdf">pdf</a>]

Content created for the Ontogenesis KnowledgeBlog meeting: <a href="http://ontogenesis.knowledgeblog.org/tag/mikel-egana-aranguren/">a paper and two reviews</a>.

Content created for the Bioinformatics KnowledgeBlog: <a href="http://bioinformatics.knowledgeblog.org/2011/07/05/publishing-information-in-the-linked-open-data-cloud/">a tutorial describing how to publish information in Linked Open Data</a>.

Basic introduction to the shell. This is a very basic tutorial on how to use the UNIX shell for non-geeks, e.g. biologists doing a bioinformatics course ;-). [<a href="basic_shell.pdf">pdf</a>]

<strong>Euskara</strong>

CAF-Elhuyar sarietara aurkeztutako artikuluak (Ez nuen saririk jaso):
<ul>
	<li>(2023) Bitcoin eta Troika: ideologia non-nahi. [<a href="CAF_Elhuyar_2022_BitCoin_Austriar_Eskola.pdf">pdf</a>]</li>
	<li>(2021) Datu zientifikoen argitalpen eraginkorra FAIR printzipioen bidez. [<a href="FAIR_printzipioak.pdf">pdf</a>]</li>
    <li>(2010) Datuen amarauna. <a href="http://eu.wikipedia.org/wiki/Datu_estekatuak">Linked Data</a>-ri buruzko artikulua [<a href="caf_elhuyar_2010_datuen_amarauna.pdf">pdf</a>]</li>
 	<li>(2009) Informazio biologikoa web semantikoaren bidez kudeatzeko diseinu patroiak. Nire tesiaren euskarazko laburpena. [<a href="mikelegana_websembio_cafelhuyar.pdf">pdf</a>]</li>
</ul>
Berria fundazioak antolatutako leihaketa batera aurkeztutako proiektua, argitaletxe digital bat sortzeko ideiekin. [<a href="mikel_egac3b1a_aranguren_txostena.pdf">pdf</a>]

<a href="https://eu.wikipedia.org/wiki/Lankide:Mikel.egana.aranguren">Euskarazko wikipedian</a> ere banabil, edukia sortzen edo hobetzen, ahal dudan heinean.

<strong>Español</strong>

Pliego técnico <a href="https://www.um.es/web/hercules/proyectos">proyecto Hércules</a>. [<a href="01678941.pdf">pdf</a>]. 

<a href="https://vimeo.com/42962819">Video divulgativo</a> sobre <a href="http://es.wikipedia.org/wiki/Datos_enlazados">Linked Data</a> para el concurso <a href="http://www.onzientzia.tv/bideoa-ikusi.asp?Kodea=332&amp;lang=ES">On Zientzia 2012</a>.

Software libre (GNU/linux) para biólogos. BioGaia 3, 2003 (<a href="http://www.cobeuskadi.es/">COBE</a>). [<a href="linuxbiologos.pdf">pdf</a>]

¿Qué puede hacer la web semántica por la biología? BioGaia 7, 2009 (<a href="http://www.cobeuskadi.es/">COBE</a>). [<a href="biogaialssw.pdf">pdf</a>]

Breve introducción a <a href="http://www.ctan.org/tex-archive/macros/latex/contrib/texshade/">TeXshade</a>, un paquete LaTeX para representar alineamientos de secuencias. [<a href="texshade_es.pdf">pdf1</a>, <a href="texshade_es_ejemplos.pdf">pdf2</a>]